dependencies.yaml

```diff
@@ -47,7 +47,7 @@ libcbv2g:
 # libiso15118
 libiso15118:
   git: https://github.com/EVerest/libiso15118.git
-  git_tag: v0.4.0
+  git_tag: dfcc1c2e03746636be8c9b1af74c0606c2cd4234
```

Comment:
Change that to the correct hash before merging

Reply:
Git_tag: v0.5.1. This release includes your changes in libiso

Reply:
Updating this branch to main and libiso to v0.5.1 should fix the build issue

Reply:
It does indeed. I have rebased, and will squash & merge once the checks are through.